### PR TITLE
Improved integrations sidebar with link to release overview + quality…

### PR DESCRIPTION
**Pull request by cskiwi**
… scale is now in bold

## Proposed change

Currently the integration sidebar has some information regarding the integration.
 
This PR includes 2 small changes:
- Added a link to the integration overview of that release 
    - Includes setting the same format of the version for each integration
- Set the quality scale in italic to improve readability 

Before change;
image

After change:
image

## Type of change

- [x] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [ ] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [ ] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

**Comment by homeassistant** [comment]
Hi @cskiwi, 

It seems you haven't yet signed a CLA. Please do so [here](https://home-assistant.io/developers/cla_sign_start/?pr=home-assistant&#x2F;home-assistant.io#24063).

Once you do that we will be able to review and accept this pull request.

Thanks!

File: source/_integrations/agent_dvr.markdown [hunk 1]
```diff
@@ -5,7 +5,7 @@ ha_category:
   - Alarm
   - Camera
 ha_config_flow: true
-ha_release: '0.110'
+ha_release: 0.110
```
Comment (balloob) [inline source/_integrations/agent_dvr.markdown]: These are strings on purpose. In numbers it will be interpreted as `0.11`

Comment (cskiwi) [reply source/_integrations/agent_dvr.markdown]: Ah, that would indeed be not the preferred result. Any suggestion to get this working with the url? can I replace the quotes?

File: source/_includes/asides/component_navigation.html [hunk 2]
```diff
@@ -15,7 +15,13 @@
   </div>
 
   <div class='section'>
-    The {{ page.name | default: page.title }} {{ page.ha_integration_type | default: "integration" }} was introduced in Home Assistant {{ page.ha_release | default: "unknown" }},
+    The {{ page.name | default: page.title }} {{ page.ha_integration_type | default: "integration" }} was introduced in Home Assistant 
+    {% if ha_release == "pre 0.7" %}
+    <a href="/integrations/#version/0.7">{{ page.ha_release | default: "unknown" }}</a>,
+    {% else %}
+    <a href="/integrations/#version/{{page.ha_release}}">{{ page.ha_release | default: "unknown" }}</a>,
```
Comment (frenck) [inline source/_includes/asides/component_navigation.html]: I am not understanding the reason to link this? What would be the added benefit?


If I look at, for example, the ESPHome integration page, why would I click on the version to view all others introduced in that release? What is the relation/added value for the user?

**Review by frenck — CHANGES_REQUESTED** [review]

File: source/_includes/asides/component_navigation.html (hunk 2 above)
Comment (cskiwi) [reply source/_includes/asides/component_navigation.html]: I can´t speak for others, but for me the following usually happens: When a new release has happened, I look into the new integrations and look for anything new that could improve my home. 

However, my memory is pretty bad. So I can't always remember that integration's name, but I do sometimes remember another integration from the same release, and thus would this provide me a quick access to the integration overview for the release version of that integration

Comment (frenck) [reply source/_includes/asides/component_navigation.html]: We already provide it from the main integration page as a filter. I think its odd, as the release itself, this specific integration has nothing to do with other integration in the same release.

I would opt to not to add it for that reason.

Comment (cskiwi) [reply source/_includes/asides/component_navigation.html]: Alright, I'll remove that link

**Pull request closed without merging**